aptos-move/framework/aptos-framework/sources/AccountUtils.move

```diff
@@ -0,0 +1,10 @@
+module AptosFramework::AccountUtils {
+    use AptosFramework::Account;
+    use AptosFramework::Coin;
+    use AptosFramework::TestCoin::TestCoin;
+
+    public(script) fun create_and_fund_account(funder: &signer, account: address, amount: u64) {
+        Account::create_account(account);
+        Coin::transfer<TestCoin>(funder, account, amount);
```

Comment:
This is pretty horrendous but I'll leave it to @movekevin to fix later :trollface:

Reply:
Yeah I have an item on my TODO list for thinking about explicit vs implicit account creation vs having a utility function like this.